sdk/src/driftClient.ts

```diff
@@ -129,6 +130,9 @@ export class DriftClient {
 	authority: PublicKey;
 	marketLookupTable: PublicKey;
 	lookupTableAccount: AddressLookupTableAccount;
+	subAccountIds?: number[];
+	includeDelegates?: boolean;
+	authoritySubaccountMap?: [string, number][];
```

Comment:
I'm thinking we only have `authoritySubaccountMap` and get rid of the `subAccountIds` on the actual DriftClient. If the user passes the subAccounts, we just default to creating the map for them given the authority?